homeassistant/components/wallbox/sensor.py

```diff
@@ -167,8 +167,12 @@ def __init__(
 
     @property
     def native_value(self) -> StateType:
-        """Return the state of the sensor."""
-        if (sensor_round := self.entity_description.precision) is not None:
+        """Return the state of the sensor. Round the value when it, and the precision property are not None."""
+        if (
```

Comment:
It would be a bit easier to read if you did instead of writing it out 3 times below, but since this is a bug fix for a patch release, less change usually better so refactoring can come in a future P

`value = self.coordinator.data[self.entity_description.key]`